src/jdk.incubator.vector/share/classes/jdk/incubator/vector/Byte128Vector.java

```diff
@@ -330,9 +330,9 @@ public final long reduceLanesToLong(VectorOperators.Associative op,
         return (long) super.reduceLanesTemplate(op, m);  // specialized
     }
 
-    @Override
     @ForceInline
-    public VectorShuffle<Byte> toShuffle() {
+    private final
+    VectorShuffle<Byte> toShuffleTemplate(AbstractSpecies<Byte> dsp) {
```

Comment:
Is it better to move this template method to the super class like other APIs?

Reply:
Yes, can be moved to super class. Done in the updated commit.